<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSourceSettingClass</class>
 <widget class="QWidget" name="ImageSourceSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Source Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white;
font-size: 11pt;
background: #323232;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,13,1">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>KaiTi</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Choose Image Source:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Camera">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>KaiTi</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_FileImage">
       <property name="font">
        <font>
         <family>KaiTi</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Local Image</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_ImageContent">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
	background: #4A4A4A;
	border-radius: 4px;
}
QPushButton:hover
{
	background: #858585;
	border-radius: 4px;
}
QPushButton:pressed
{
	background: #A3A3A3;
	border-radius: 4px;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Save">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background: #3A71DE;
	border-radius: 4px;
}
QPushButton:hover
{
	background: #274B94;
	border-radius: 4px;
}
QPushButton:pressed
{
	background: #172D59;
	border-radius: 4px;
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
